starters/default/src/templates/using-dsg.js

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { Link } from "gatsby"
+
+import Layout from "../components/layout"
+import Seo from "../components/seo"
+
+const UsingDSG = () => (
+  <Layout>
+    <Seo title="Using DSG" />
+    <h1>Hello from a DSG Page</h1>
+    <p>This page is not created until requested by a user.</p>
+    <p>
+      To learn more, head over to our{" "}
+      <a href="https://v4.gatsbyjs.com/docs/reference/rendering-options/deferred-static-generation/">
```

Comment:
Once this is ready for GA, we will likely want to use the main domain instead of the `v4.` subdomain